contrib/caldav/calcurse-caldav.py

```diff
@@ -508,6 +508,12 @@ def run_hook(name):
 verbose = args.verbose
 debug = args.debug
 
+# Read environment variables
+try:
+    password = os.environ['CALCURSE_PASSWORD']
+except KeyError:
+    password = None
+
```

Comment:
I think it is much cleaner (and shorter) to use `password = os.getenv('CALCURSE_PASSWORD')` here.